src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -35,32 +35,35 @@
     /**
      * Max size of the stacks.
      */
-    private static final int MAX_STACK_SIZE = 64;
+    private static final int                    MAX_STACK_SIZE   = 64;
     /**
      * The returned slot if a slot hasn't been found.
      */
-    private static final int NO_SLOT        = -1;
+    private static final int                    NO_SLOT          = -1;
     /**
      * Size of the hotbar.
      */
-    private static final int HOTBAR_SIZE    = 0;
+    private static final int                    HOTBAR_SIZE      = 0;
     /**
      * The main inventory.
      */
-    private final NonNullList<ItemStack> mainInventory    = NonNullList.<ItemStack>withSize(36, ItemStack.EMPTY);
+    private final        NonNullList<ItemStack> mainInventory    = NonNullList.<ItemStack>withSize(36, ItemStack.EMPTY);
     /**
      * The armour inventory.
      */
-    private final NonNullList<ItemStack> armorInventory   = NonNullList.<ItemStack>withSize(4, ItemStack.EMPTY);
+    private final        NonNullList<ItemStack> armorInventory   = NonNullList.<ItemStack>withSize(4, ItemStack.EMPTY);
     /**
      * The off-hand inventory.
      */
-    private final NonNullList<ItemStack> offHandInventory = NonNullList.<ItemStack>withSize(1, ItemStack.EMPTY);
+    private final        NonNullList<ItemStack> offHandInventory = NonNullList.<ItemStack>withSize(1, ItemStack.EMPTY);
     private final List<NonNullList<ItemStack>> allInventories;
     /**
      * The index of the currently held item (0-8).
      */
     public        int                          currentItem;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make currentItem a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)